bin/tcp_server.ml

```diff
@@ -0,0 +1,65 @@
+open Lwt
```

Comment:
It's not good style to open this library specifically, as it has a lot of toplevel functions in scope. Usually one just opens `Lwt.Infix` or `Lwt.Syntax` to get the bind/map infix/let-operators by themselves.

Some libraries are made to be opened though - e.g. Daniel Bünzli libraries like `Gg`, `Bos`, `Vg`. This (sadly) depends on the library in question.